packages/server/lib/graphql/ServerActions.ts

```diff
@@ -105,17 +108,34 @@ export class ServerActions extends BaseActions {
     fs.writeFileSync(path.resolve(project.projectRoot, configFilename), code)
   }
 
-  async initializeOpenProject (args: LaunchArgs, options: OpenProjectLaunchOptions) {
-    await openProject.create(args.projectRoot, args, options)
+  async initializeOpenProject (args: LaunchArgs, options: OpenProjectLaunchOptions, browsers: FoundBrowser[]) {
+    await openProject.create(args.projectRoot, args, options, browsers)
+    if (args.testingType === 'e2e') {
+      this.ctx.activeProject!.setE2EPluginsInitialized(true)
```

Comment:
We should lint against this for sure

In this case we likely just want a null check/throw error, since you should not be able to progress to this part of the UI without an active project previously set